demo/tsconfig.json

```diff
@@ -23,6 +23,7 @@
         "baseUrl": ".",
         "paths": {
             "*": [
+                "./node_modules/tns-core-modules/*",
```

Comment:
The full imports will get resolved faster during build time, but the difference will be negligible unless there are hundreds of thousands imports from `tns-core-modules`. It's just because TS will first try to find the import as-is and then go through each of the path mappings.